cumulus/polkadot-parachain/src/main.rs

```diff
@@ -14,11 +14,66 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Cumulus test parachain collator
+//! Polkadot parachain-omni-node.
 
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
+pub(crate) const EXAMPLES: &str = color_print::cstr!(
+	r#"<bold><underline>Examples:</></>
+
+   <bold>polkadot-parachain-omni-node --chain para.json --sync warp -- --chain relay.json --sync warp</>
+        Launch a warp-syncing full node of a given para's chain-spec, and a given relay's chain-spec.
+
+	<green><italic>The above approach is the most flexible, and the most forward-compatible used to spawn an omni-node.</></>
+
+	You can find the chain-spec of some networks in:
+
+	https://paritytech.github.io/polkadot_network_directory
```

Comment:
New version of this is out, please replace it cc @PierreBesson 

https://paritytech.github.io/chainspecs/

Reply:
Done